packages/ecs/src/QueryFunctions.tsx

```diff
@@ -80,7 +80,7 @@ export function removeQuery(queryOrTerms: ReturnType<typeof defineQuery> | bitEC
   }
 }
 
-export const query = (queryTerms: bitECS.QueryTerm[]) => bitECS.query(HyperFlux.store, queryTerms)
+export const query = (queryTerms: bitECS.QueryTerm[]) => bitECS.query(HyperFlux.store, queryTerms) as readonly Entity[]
```

Comment:
can we add a second optional param here that defualts to the simulation layer, and include it in the query?

Reply:
Can do that in a separate PR